lib/src/utils/serialization_utils.dart

```diff
@@ -761,7 +832,7 @@ class SerializationUtils {
     });
   }
 
-  static Uint8List serializeToken<T>(ParsedToken token, Uint8List Function(T) serializeTokenAmountFn) {
+  static Uint8List serializeToken(ParsedToken token, Uint8List Function(BigInt) serializeTokenAmountFn) {
```

Comment:
The second argument `Uint8List Function(BigInt) serializeTokenAmountFn` is not needed at all. Always `SerializationUtils.int64ToBuf` is used.

Reply:
done